src/services/codefixes/importFixes.ts

```diff
@@ -644,6 +665,20 @@ namespace ts.codefix {
             Debug.fail("Either the symbol or the JSX namespace should be a UMD global if we got here");
         }
 
+        const allowSyntheticDefaultImports = getAllowSyntheticDefaultImports(compilerOptions);
+
+        // Prefer to import as a synthetic `default` if available.
+        if (allowSyntheticDefaultImports) {

```

Comment:
@weswigham this is another place for your new change in https://github.com/Microsoft/TypeScript/pull/19675 to be plugged in. we would like to always show the `default` import in this case and never fallback to `import  .. = require`

Reply:
Roger. Thank you!